vault/core.go

```diff
@@ -2424,6 +2425,8 @@ func (c *Core) SetLogLevel(level log.Level) {
 // SetConfig sets core's config object to the newly provided config.
 func (c *Core) SetConfig(conf *server.Config) {
 	c.rawConfig.Store(conf)
+	bz, _ := json.Marshal(c.SanitizedConfig())
```

Comment:
Why ignore err?

Reply:
We'd have to change the API for that. Alternatively, we could not log at all upon error.

Reply:
Why not log the error when it happens, instead of the config?